api/yangCatalogApi.py

```diff
@@ -327,6 +330,15 @@ def create_response_with_yangsuite_link(self, response):
 application.config["OIDC_CALLBACK_ROUTE"] = "/api/admin/ping"
 application.config["OIDC_SCOPES"] = ["openid", "email", "profile"]
 application.config["OIDC_ID_TOKEN_COOKIE_NAME"] = "oidc_token"
+application.config["SQLALCHEMY_DATABASE_URI"] = URL.create('mysql', username=yc_gc.dbUser, password=yc_gc.dbPass,
```

Comment:
Please add a TODO here that all global config variables need to be taken into flask application.config variable and this init_app part of the code should be moved there as well. For now we will keep it in here. Create and issue for that as well before and provide in your todo reference to the issue number something like backend issue #XYZ